Data/Sector/Hong-Kong-Sector-File.sct

```diff
@@ -145,6 +145,8 @@ E113.55.16.824
 #define APP 5865472
 #define DEP 5865580
 #define ConstructionArea 4342338
+#define ControlledCTR 16753920
+#define UncontrolledUCARA 6749952
```

Comment:
You can sort of see the CTR subdivisions here:
![image](https://github.com/vatsimhk/Hong-Kong-Sector-Package/assets/89831445/83b9ac48-fdcd-4623-8ee6-55f3eb21efd3)

They're the same colour as the sector boundaries - so I set them to the sector boundary colour in the sct

Reply:
Looks clean but then this will remove the distinction between controlled and uncontrolled areas, even if controllers should have them memorised, what do you think @Syntax505 @UraniumLover

Reply:
Looks much cleaner to me, so I'm not opposed to the change. Up to @UraniumLover to decide

Reply:
lysm